src/js/utils/ResourcesUtil.js

```diff
@@ -6,7 +6,7 @@ const DefaultResourceTypes = {
   cpus: { label: "CPU", colorID: 0 },
   mem: { label: "Memory", colorID: 6 },
   disk: { label: "Disk", colorID: 3 },
-  gpus: { label: "GPU", colorID: 0 }
+  gpus: { label: "GPU", colorID: 8 }
```

Comment:
Just curious, why do we use numbers for the color ID instead of just a string with the color name? That feels more readable both here and in the CSS

Reply:
🤷‍♂️